rir/src/compiler/native/lower_function_llvm.cpp

```diff
@@ -1546,10 +1560,11 @@ llvm::Value* LowerFunctionLLVM::depromise(Value* v) {
 }
 
 void LowerFunctionLLVM::compileRelop(
-    Instruction* i,
+    BinopKind kind, Instruction* i,
```

Comment:
I know nothing to do with your PR, but couldn't we find out the binop kind from the instruction?